docs/paper/dev/getting-started/plugin-yml.md

```diff
@@ -0,0 +1,192 @@
+- `author: PaperMC`
+- `authors: [PaperMC, SpigotMC, Bukkit]`
+
+### website
+
+The website of the plugin. This is useful for linking to a GitHub repository or a plugin page.
+- `website: https://papermc.io`
+
+### api-version
+
+The version of the Paper API that your plugin is using. This doesn't include the minor version.
+Servers with a version lower than the version specified here will refuse to load the plugin.
+The valid versions are 1.13 - 1.19.
+- `api-version: 1.19`
+
+:::info
+
+If this is not specified, the plugin will be loaded as a legacy plugin and a warning will be printed to the console.
+
+:::
+
+### load
+
+This tells the server when to load the plugin. This can be `STARTUP` or `POSTWORLD`.
+- `load: STARTUP`
+
+### prefix
+
+The prefix of the plugin. This is what will be displayed in the log instead of the plugin name.
+- `prefix: EpicPaperMCHypePlugin`
+
+### libraries
+
+:::warning
+
+There is debate over whether this is against maven central's TOS. This may be removed in the future.
+
+:::
+
+This is a list of libraries that your plugin depends on. These libraries will be downloaded from the Maven central repository and added to the classpath.
+This removes the need to shade and relocate the libraries.
+
+```yaml
+libraries:
+  - com.google.guava:guava:30.1.1-jre
+  - com.google.code.gson:gson:2.8.6
+```
+
+### permissions
+
+This is a list of permissions that your plugin uses. This is useful for plugins that use permissions to restrict access to certain features.
+```yaml
+permissions :
+  - permission.node:
+        description: "This is a permission node"
+        default: op
+        children:
+            - permission.node.child: true
+  - another.permission.node:
+        description: "This is another permission node"
+        default: not op
+```
+
+The description is the description of the permission node. This is what will be displayed in the permissions list.
+The default is the default value of the permission node. This can be `op`/`not op`, or `true`/`false`. This defaults to `op`.
+Each permission node can have children. When set to `true` it will inherit the parent permission.
+
+## Commands
+
+This is a list of commands that your plugin uses. This is useful for plugins that use commands to provide features.
+```yaml
+commands:
+  - command:
+        description: "This is a command"
+        usage: "/command <arg>"
+        aliases: [cmd, command]
+        permission: permission.node
+        permission-message: "You do not have permission to use this command"
+```
+
+- The __description__ is the description of the command. This gives a brief description of what the command does.
+- The __usage__ is the usage of the command. This is what will be displayed when the player uses `/help <command>`.
+- The __aliases__ are a list of aliases that the command can be used with. This is useful for shortening the command.
+- __permission__ is the permission node that the player needs to use the command.
```

Comment:
maybe add something here that players will not see commands they dont have permission for

Reply:
Yep, that's something I forgot to be honest 👍